css/style.css

```diff
@@ -15,5 +14,195 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+h1 {
+  margin: auto;
+  margin-top: auto;
+  font-size: 40px;
```

Comment:
As we discussed last week, using pixels for font sizes is fine for the moment. rem's are better for accessibility as the font size will change when the user updates their text size in their browser settings. 1 rem equals 16px in all main browsers.